minidump-processor/src/stackwalker/mod.rs

```diff
@@ -198,6 +198,7 @@ where
     CallStack {
         frames,
         info,
+        thread_id: 0,
```

Comment:
This is just a placeholder. It's overwritten after `stackwalker::walk_stack` in the processor.